conversion.md

```diff
@@ -0,0 +1,89 @@
+# Conversion to OCI Runtime Configuration
+
+When extracting an OCI Image into an [OCI Runtime bundle][oci-runtime-bundle], two components of the orthogonal components of the extraction are relevant:
+
+1. Extraction of the root filesystem from the set of [filesystem layers](layers.md).
+2. Conversion of the [image configuration blob](config.md) to an [OCI Runtime configuration blob][oci-runtime-config].
+
+This section defines how to convert an `application/vnd.oci.image.config.v1+json` blob to an [OCI runtime configuration blob][oci-runtime-config] (the latter component of extraction).
+The former component of extraction is defined [elsewhere](layers.md) and is orthogonal to configuration of a runtime bundle.
+
+[oci-runtime-bundle]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/bundle.md
+[oci-runtime-config]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/config.md
+
+## Verbatim Fields
+
+Certain image configuration fields have an identical counterpart in the runtime configuration.
+A compliant configuration converter MUST extract the following fields verbatim to the corresponding field in the generated runtime configuration:
+
+| Image Field         | Runtime Field   | Notes |
+| ------------------- | --------------- | ----- |
+| `architecture`      | `platform.arch` |       |
+| `os`                | `platform.os`   |       |
+| `Config.WorkingDir` | `process.cwd`   |       |
+| `Config.Env`        | `process.env`   | 1     |
+| `Config.Entrypoint` | `process.args`  | 2     |
+| `Config.Cmd`        | `process.args`  | 2     |
+| `Config.Labels`     | `annotations`   | 3     |
+
+1. The converter MAY *prepend* additional environment variables to `process.env`, but it MUST NOT *append* values to the environment.
+2. If both `Config.Entrypoint` and `Config.Cmd` are specified, the converter MUST append the value of `Config.Cmd` to the value of `Config.Entrypoint` and set `process.args` to that combined value.
+3. See the section on [annotations](#annotations) for caveats.
+
+## Parsed Fields
+
+Certain image configuration fields have a counterpart that must first be translated.
+A compliant configuration converter SHOULD parse all of these fields and set the corresponding fields in the generated runtime configuration:
+
+| Image Field         | Runtime Field    | Notes |
+| ------------------- | ---------------  | ----- |
+| `Config.User`       | `process.user.*` |       |
+
+### `Config.User`
+
+When converting the value of `Config.User` a converter SHOULD resolve the user information using a method appropriate for the container's context.
+For Linux based systems this would involve parsing `/etc/passwd` from the extracted container's root filesystem to determine the values of `process.user.uid` and `process.user.gid`.
```

Comment:
The config docs allow [both numeric and named user/group values][1], and only mention `/etc/passwd` in [the context of default groups][2].  So I think we need to weaken this line to say “MAY involve parsing `/etc/passwd` and `/etc/group`” to allow for:

* Configs using numberic uid/gids (where there is no need for a lookup)
* Agents using other NSS backends (e.g. LDAP), since the config spec doesn't require a particular NSS backend.

[1]: https://github.com/opencontainers/image-spec/blame/v1.0.0-rc3/config.md#L79
[2]: https://github.com/opencontainers/image-spec/blame/v1.0.0-rc3/config.md#L80